index.html

```diff
@@ -1,34 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
Please add steps to install the project locally in your README.

Reply:
Added steps to install locally